w3cdatalayer/module.go

```diff
@@ -17,3 +18,8 @@ func (m *Module) Configure(injector *dingo.Injector) {
 	flamingo.BindTemplateFunc(injector, "w3cDatalayerService", new(templatefunctions.W3cDatalayerService))
 	flamingo.BindEventSubscriber(injector).To(application.EventReceiver{})
 }
+
+// CueConfig schema and configuration
+func (m *Module) CueConfig() string {
+	return `w3cDatalayer: {	hashEncoding: string | *"base64url" }`
```

Comment:
prefix with `commerce: ` to follow the namespacing

Reply:
Updated.